src/valour.js

```diff
@@ -95,6 +99,18 @@ class Valour {
     let result = this.getResult(name);
     return !Object.keys(result).some((key) => !result[key].valid);
   }
+
+  setValidationState(name, data) {
+    let form = this.getForm(name);
+    Object.keys(form).forEach((key) => {
+      if (data[key] === undefined) {
+        return;
+      }
+      let { valid, messages } = data[key];
+      form[key].setState(valid, messages);
+    });
+    this.runCallbacks(name);
+  }
```

Comment:
I think it would make sense for us to add the opportunity to do this with your config object in `register`, as well. That way you could do all your configuration and initialization in one place, if that's your approach.